packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -313,6 +323,7 @@ class AdaptiveScaffold extends StatefulWidget {
     return Builder(
       builder: (_) {
         return BottomNavigationBar(
+          type: BottomNavigationBarType.fixed,
```

Comment:
Will this remove the animation of the bar?

Reply:
Type shall be considered from theme.